### ISSUE TITLE
Bug: prod environment isn't launching

### ISSUE THREAD

**Issue by KFearsoff** [catebi/FreeganBot#85]
If `ENV` is set to `PROD`, the bot crashes on startup. This is the reason: https://github.com/catebi/FreeganBot/blob/a94451bed636f5b5edfdce7313ec98dd55fb211d/utils/env_processor.py#L87-L89

Additionally, `CATEBI_API_LINK` provided in `env.example` is wrong: the correct API path is `https://api.catebi.ge/api/freegan/`, not `https://api.catebi.ge/api/`: https://github.com/catebi/FreeganBot/blob/a94451bed636f5b5edfdce7313ec98dd55fb211d/env.example#L24

The most basic solution is to change this line to:

```python
response = requests.get(csl.catebi_api() + 'getdonationchats')
```

List of items:
- Fix the issue
- Make sure the URLs are concatenated correctly; that is, both `https://api.catebi.ge/api/freegan/` and `https://api.catebi.ge/api/freegan` should work the same way. There are probably libraries for that
- Make sure the API is used correctly; there are also `https://api.catebi.ge/api/Freegan/SaveMessage` and `https://api.catebi.ge/api/freegan/SaveReaction` endpoints

**Comment by KFearsoff** [comment]
@lejafo @MSenso

**Comment by lejafo** [comment]
yes. good point. i suppose we need to put this request to CatebiEndpoints
